seq2science/rules/qc.smk

```diff
@@ -854,10 +854,10 @@ def get_qc_files(wildcards):
         qc["files"].update(files)
 
     # DESeq2 all contrast plots
-    if "get_contrasts" in globals():
+    if "DE_contrasts" in globals():
```

Comment:
should we maybe make `DE_contrasts` just an empty list as default? Not sure what is better here

Reply:
Although maybe checking if it exists is the easier solution indeed

Reply:
using a global variable or checking if a variable exists. seems like chosing the lesser evil :p